Client/Frontend/Settings/AppSettingsTableViewController.swift

```diff
@@ -143,4 +148,21 @@ class AppSettingsTableViewController: SettingsTableViewController {
 
         return super.tableView(tableView, heightForRowAtIndexPath: indexPath)
     }
+
+    override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        // Hide the top border for the top section to avoid having a double line at the top
+        if !profile.hasAccount() && section == 2 {
+            let headerView = tableView.dequeueReusableHeaderFooterViewWithIdentifier(SectionHeaderIdentifier) as! SettingsTableSectionHeaderFooterView
+            let sectionSetting = settings[section]
+
+            if let sectionTitle = sectionSetting.title?.string {
```

Comment:
Nit: You don't need to unwrap the string from sectionSetting.title? since the titleLabel's text is optional anyways. We can rewrite this as:

`headerView.titleLabel.text = sectionSetting.title?.string`